pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a placeholder:
```

Comment:
```suggestion
- Open an interactive prompt for the user to input a string with a specific placeholder:
```

Reply:
Hey, Shouldn't this line be
```suggestion
- Open an interactive prompt for the user to input a commit message:
``` 
[Reference](https://github.com/charmbracelet/gum#readme)
![image](https://user-images.githubusercontent.com/26346867/183278051-adcfe256-41d9-4525-ba7f-cff1ba11c4de.png) 
![image](https://user-images.githubusercontent.com/26346867/183278201-9a13595d-d1d2-4ce0-8d93-3d28550e448b.png)

Reply:
> Hey, Shouldn't this line be
> 
> [Reference](https://github.com/charmbracelet/gum#readme) ![image](https://user-images.githubusercontent.com/26346867/183278051-adcfe256-41d9-4525-ba7f-cff1ba11c4de.png) ![image](https://user-images.githubusercontent.com/26346867/183278201-9a13595d-d1d2-4ce0-8d93-3d28550e448b.png)

No, since the command doesn't need to be used for a commit message or whatever, that is just an example they provide.